airflow/auth/managers/base_auth_manager.py

```diff
@@ -367,7 +367,7 @@ def _is_permitted_dag_id(method: ResourceMethod, methods: Container[ResourceMeth
             if _is_permitted_dag_id("GET", methods, dag_id) or _is_permitted_dag_id("PUT", methods, dag_id)
         }
 
-    def get_permitted_menu_items(self, menu_items: list[MenuItem]) -> list[MenuItem]:
+    def filter_permitted_menu_items(self, menu_items: list[MenuItem]) -> list[MenuItem]:
```

Comment:
Isn't that backwards-incompatible? Or we do not care too much at this stage?

Reply:
You're right. I dont think anybody use that interface yet so I thought it was the good time to do it. I guess your statement "Or we do not care too much at this stage?" is correct :)

Reply:
I can do that :)

Reply:
Done